src/test/java/org/folio/rest/api/ItemEffectiveCallNumberComponentsTest.java

```diff
@@ -288,6 +287,15 @@ private IndividualResource createHoldingsWithPropertySetAndInstance(
     return holdings;
   }
 
+  /**
+   * ItemStorageTest.nod with random id and without barcode
+   */
+  private JsonObject nod(UUID holdingsRecordId) {
+    JsonObject nod = ItemStorageTest.nod(holdingsRecordId);
+    nod.remove("barcode");
+    return nod;
+  }
```

Comment:
Given the intention is this item should not have a barcode, I suggest this be described in the method name rather than a comment (meaning a consumer does not need to check the comments / implementation).

```suggestion
  static JsonObject nodWithNoBarcode(UUID holdingsRecordId) {
```